### PR TITLE
AHTI-217 | Implement Breadcrumb component

### PR DESCRIPTION
**Pull request by ahie**

### CODE REVIEW

File: src/common/ui-components/Breadcrumb/Pill.tsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  readonly category: string;
+  readonly name: string;
+  onClose(): void;
+}
+
+const Pill: React.FC<PillProps> = ({ category, name, onClose }) => {
+  return (
+    <div className={styles.pill}>
+      <div>
+        <IconLocation className={styles.smallIcon} />
```
Comment (ahie) [inline src/common/ui-components/Breadcrumb/Pill.tsx]: Should be replace with `<CategoryIcon>` which is added in the `ListView` PR.

Comment (marharyta) [reply src/common/ui-components/Breadcrumb/Pill.tsx]: That PR is merged, go ahead

Comment (marharyta) [reply src/common/ui-components/Breadcrumb/Pill.tsx]: Also, your PRs are awesome

File: .stylelintrc.json [hunk 2]
```diff
@@ -26,7 +26,7 @@
     "rule-empty-line-before": "always",
     "selector-no-vendor-prefix": true,
     "selector-max-universal": 0,
-    "selector-max-type": 0,
+    "selector-max-type": 1,
```
Comment (marharyta) [inline .stylelintrc.json]: Good catch!

File: src/common/ui-components/Breadcrumb/Pill.tsx (hunk 1 above)
Comment (ahie) [reply src/common/ui-components/Breadcrumb/Pill.tsx]: Done. And thanks 👍

**Pull request merged**